src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -144,75 +147,62 @@ def export(
                         resp.content.decode("utf-8")))
             except Exception as ex:
                 self.__logger.warning("Failed to export metrics: %s", ex)
-                return MetricsExportResult.FAILURE
+                return MetricExportResult.FAILURE
 
-        return MetricsExportResult.SUCCESS
+        return MetricExportResult.SUCCESS
 
-    def _to_dynatrace_metric(self, metric: MetricRecord):
+    def _to_dynatrace_metric(self, metric: Metric):
         try:
-            attrs = dict(metric.labels)
-            if isinstance(metric.aggregator, aggregate.SumAggregator):
-                if not self._is_delta_export:
-                    self.__logger.info(
-                        "Received cumulative value which is currently"
-                        " not supported, using gauge instead.")
-                    # TODO: implement and use a Cumulative-to-Delta converter
+            attrs = dict(metric.attributes)
+            if isinstance(metric.point, Sum):
+                if isinstance(metric.point.value, float):
+                    return self._metric_factory.create_float_counter_delta(
```

Comment:
Do we really need this distinction between float and int? I think it should be fine to export always as float, no? @pirgeo

Reply:
I think it is valid to have the destinction as the only difference to before is that the distinction of value-type by instrument is now implicit based on the values used with it, not explicit on instrument creation.

Reply:
And the underlying `ToString` conversion runs the same code for both currently anyway. If we ever want to treat floats differently on the wire its easy, since there is a clear separation all the way through!